### PR TITLE
Entity relationship graph post

### PR DESCRIPTION
**Pull request by bricel**
@amitaibu - Please review.

### CODE REVIEW

**Comment by bricel** [comment]
https://www.drupal.org/sandbox/bricel/2654176

File: app/_posts/2016-01-06-entity-relations-graph.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+author: bricel
+---
+
+{% include setup %}
+
+Many times we are challenges with the maintenance of existing projects that were developed by other agencies, or a new
+developer arrives and we need to quickly bring him on board, the complexity of legacy projects can be very high and there
+are many aspects that define a Drupal project, contrib module, custom modules, themes and so one. One way we like to look at a project before diving into the code is through the different entities and bundles, so I figures if we could generates a graph
+with all the bundles, entities, and relations, this would be very helpfull.
+
```
Comment (amitaibu) [inline app/_posts/2016-01-06-entity-relations-graph.md]: add image above the `more `

File: app/_posts/2016-01-06-entity-relations-graph.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+or a simplified graph with no fields at
+
+```gherkin
+http://yoursite.com/admin/entity-relations/node/false
+```
+
+
+## Examples
+Simplified node graph
+<img class="img-responsive" src="https://cloud.githubusercontent.com/assets/165644/12092755/ad4bb60e-b307-11e5-904f-a75ee8db7b5c.png">
```
Comment (amitaibu) [inline app/_posts/2016-01-06-entity-relations-graph.md]: (btw, I just remembered the reason - our grunt task also optimizes the images). Anyway, I'll take care of it for you.

**Comment by amitaibu** [comment]
@niryariv please review

File: app/_posts/2016-01-24-entity-relations-graph.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+---
+
+
+{% include setup %}
+
+We are often challenged with the maintenance of existing projects that were developed by other agencies, or a new developer arrives and we need to quickly bring them on board. The complexity of legacy projects can be very high and the risk of breaking existing logic is something we want to avoid.
+
+One way we like to look at a project before diving into the code is through its data structure. The different entities and their relations can tell us a lot about the business logic of the site and its internal logic. We assumed that if we could _easily_ generate a graph with all the bundles, entities, and their relations this complex task would be easier.
+
+Having done this for a while now, I believe our assumption was right. For example it's much easier to look at the following graph and understand that `Work session`, `Time tracking`, `Payment`, and `Github Issue` bundles are pointing at a `Project`, which in turn points to an `Account` bundle. `Github Issue` can also reference itself.
```
Comment (amitaibu) [inline app/_posts/2016-01-24-entity-relations-graph.md]: @bricel i'm adding a link to productivity repo here

**Comment by amitaibu** [comment]
Merged, thanks.

http://www.gizra.com/content/entity-relationships/

**Pull request merged**